lib/monitor/measure.go

```diff
@@ -36,124 +75,142 @@ func ConnectSink(addr string) error {
 		return err
 	}
 	dbg.Lvl3("Connected to sink:", addr)
+	sink = addr
 	connection = conn
 	encoder = json.NewEncoder(conn)
 	return nil
 }
 
-// Only sends a ready-string
-func Ready(addr string) error {
-	if encoder == nil {
-		dbg.Lvl3("Connecting to sink", addr)
-		err := ConnectSink(addr)
-		if err != nil {
-			return err
-		}
+func StopSink() {
```

Comment:
exported method StopSink should have comment or be unexported. Says your human linter.